src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -883,6 +886,10 @@ export class PatientViewPageStore {
             this.studiesForSamplesWithoutCancerTypeClinicalData,
             this.samplesWithoutCancerTypeClinicalData);
     }
+    
+    @computed get genePanelData() {
```

Comment:
this isn't a good name because it hides the fact that it's data for a single selected genePanel.   better just to get rid of this computed and de-reference the data where you need it.

Reply:
I agree that it's not a good name but I think the computed makes sense for this. name should be `modalSelectedGenePanel` or something like that